### ISSUE TITLE
Data via API

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/blocked-org-uk#164]
Data isn't currently available off the API links.

Whatever is available should not include categorisation data scraped from the filters.

It also should not include negative results against domains obtained via registries.

**Comment by dantheta** [comment]
Yep, we pulled those files a while back.  We can reinstate them at some point, since we've been very careful to record the source of each of the URL lists.  It's possible to separate out the sources to only provide bulk data from unencumbered sources.  

I've removed the links for now.

**Comment by dantheta** [comment]
Suggest reinstating sanitised downloads after launch. There are no dangling references to the files in the copy.

**Comment by JimKillock** [comment]
See also #237 – display API calls on pages
